src/ui/components/Library/Team/View/Tests/Overview/TestOverviewContainerContextType.tsx

```diff
@@ -0,0 +1,8 @@
+import { ReactNode, createContext } from "react";
+type TestOverviewContainerContextType = {};
+
+export const TestOverviewContext = createContext<TestOverviewContainerContextType>(null as any);
+
+export function TestOverviewContainer({ children }: { children: ReactNode }) {
+  return <TestOverviewContext.Provider value={{}}>{children}</TestOverviewContext.Provider>;
+}
```

Comment:
This component doesn't really add much value. Since it's only used in one place (`TestOverviewPage`) can you just inline the context provider there?

Reply:
Yep — deleted this structure so we can just use the `TestOverviewContent` component directly